### PR TITLE
set default values for openfaas parameters

### PR DESCRIPTION
**Pull request by developer-guy**
Signed-off-by: Batuhan Apaydın <batuhan.apaydin@trendyol.com>

**What type of PR is this?**

> Uncomment one (or more) `/kind <>` lines:

> /kind feature

**Any specific area of the project related to this PR?**

> Uncomment one (or more) `/area <>` lines:

> /area falcosidekick-chart

**What this PR does / why we need it**:

We set default values for the OpenFaaS output type to help developers to remember the values that they need to set by reducing the number of parameters that should be set.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Checklist**

- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] CHANGELOG.md updated

### CODE REVIEW

**Comment by developer-guy** [comment]
PTAL @Issif 🙏

File: falcosidekick/CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,13 @@ numbering uses [semantic versioning](http://semver.org).
 
 Before release 0.1.20, the helm chart can be found in `falcosidekick` [repository](https://github.com/falcosecurity/falcosidekick/tree/master/deploy/helm/falcosidekick).
 
+## 0.3.3
+
+### Fixes
+
+* Set default values for OpenFaaS output type parameters
+* Fixes on documentations
```
Comment (Issif) [inline falcosidekick/CHANGELOG.md:13]:
```suggestion
* Set default values for `OpenFaaS` output type parameters
* Fixes of documentation
```

Comment (developer-guy) [reply falcosidekick/CHANGELOG.md]: done

File: falcosidekick/CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,13 @@ numbering uses [semantic versioning](http://semver.org).
 
 Before release 0.1.20, the helm chart can be found in `falcosidekick` [repository](https://github.com/falcosecurity/falcosidekick/tree/master/deploy/helm/falcosidekick).
 
+## 0.3.3
+
+### Fixes
+
+* Set default values for OpenFaaS output type parameters
+* Fixes of documentations
```
Comment (Issif) [inline falcosidekick/CHANGELOG.md:13]:
```suggestion
* Set default values for `OpenFaaS` output type parameters
* Fixes of documentation
```

Comment (developer-guy) [reply falcosidekick/CHANGELOG.md]: ok, now its done 👍

**Review by Issif — APPROVED** [review]

**Pull request merged**